### PR TITLE
refactor: status/events

### PR DESCRIPTION
**Pull request by ErinHales**
- bug: reference error, fetch undefined (cannot get hosted configs)
 - feature: use indexer to get events

## Motivation

Ability to get status easily and accurately is important for partners trying to integrate with Nomad. We've moved away from this code internally and have not maintained it. We should allow use of indexer for specific queries using the sdk.

## Solution

 1. delete old events code and refactor initializer functions
 2. make an internal `_events()` function that updates a tx's events to the latest and stores them in the cache
 3. make other getter functions for specific events (relay, update, process), call `_events()` only when needed

TODO:
 - [ ] update tests
 - [x] update sdk-bridge
 - [x] check for other downstream changes (indexer, local-env, etc)

## PR Checklist

- [ ] Added Tests
- [ ] Updated Documentation
- [ ] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/sdk-bridge/src/bridgeEvents.ts [hunk 1]
```diff
@@ -1,29 +0,0 @@
-import { TypedEvent } from '@nomad-xyz/contracts-core';
```
Comment (Imti) [inline packages/sdk-bridge/src/bridgeEvents.ts:1]: 🥳 Yay deleted code

File: packages/sdk/package.json [hunk 2]
```diff
@@ -35,7 +35,10 @@
     "@nomad-xyz/contracts-core": "workspace:^",
     "@nomad-xyz/multi-provider": "workspace:^",
     "axios": "^0.27.2",
```
Comment (Imti) [inline packages/sdk/package.json:37]: I'd prefer just keeping cross-fetch over axios if that one works better for us. Not a big deal for now tho. Noticed we only use axios in `getProof()`

**Comment by ErinHales** [comment]
Just remembered I need to update the changelog

**Review by Imti — APPROVED** [review]
Big refactor, awesome 🌳

File: packages/sdk/package.json (hunk 2 above)
Comment (ErinHales) [reply packages/sdk/package.json]: I just decided to include those changes in this PR, so I'm not mixing code with a release PR

**Pull request merged**